uisdk/src/main/java/com/karhoo/uisdk/screen/booking/checkout/component/views/CheckoutView.kt

```diff
@@ -466,6 +481,58 @@ internal class CheckoutView @JvmOverloads constructor(context: Context,
         }
     }
 
+    private fun bindComments(comments: String?){
+        bookingCheckoutCommentView.setActionIcon(R.drawable.kh_uisdk_ic_checkout_comments)
+
+        bookingCheckoutCommentView.setTitle(context.getString(R.string.kh_uisdk_checkout_comments_title))
+        comments?.let {
+            bookingComments = it
+            if(it.isEmpty())
+                bookingCheckoutCommentView.setSubtitle(context.getString(R.string.kh_uisdk_checkout_comments_subtitle))
+            else
+                bookingCheckoutCommentView.setSubtitle(it)
+        }?: kotlin.run {
+            bookingCheckoutCommentView.setSubtitle(context.getString(R.string.kh_uisdk_checkout_comments_subtitle))
+        }
+    }
+
+    private fun showCommentsDialog(){
+        val commentBottomSheet = CheckoutCommentBottomSheet().apply {
+            onCommentsChanged = {
+                bindComments(it)
+            }
+            bookingComments.isNotEmpty().let {
+                initialComments = bookingComments
+            }
+        }
+        commentsListener?.invoke(commentBottomSheet)
+    }
+
+    private fun bindTravelDetails(){
+        val capabilities = presenter.getCurrentQuote()?.fleet?.capabilities
+        if(isPrebook){
```

Comment:
I think we should do the logic below inside the CheckoutPresenter and have it call view.bindTravelDetails(visibility, actionIcon, title, subtitle)